Sparkle/SUAppcastItem.m

```diff
@@ -55,6 +55,11 @@ - (BOOL)isCriticalUpdate
     return [self.propertiesDictionary[SUAppcastElementTags] containsObject:SUAppcastElementCriticalUpdate];
 }
 
+- (BOOL)isInformationOnlyUpdate
+{
+	return (([self infoURL] != nil) && ([self fileURL] == nil));
```

Comment:
This style is mixed in Sparkle's project (and before in this file uses != nil too). Personally I think implicit checking can potentially remove type safety in certain circumstances.. Just my 2 cents.